tmt/steps/provision/podman.py

```diff
@@ -19,6 +19,7 @@
 DEFAULT_USER = "root"
 DEFAULT_PULL_ATTEMPTS = 5
 DEFAULT_PULL_INTERVAL = 5
+STOP_TIME = "1"  # podman default is 10 seconds
```

Comment:
I'd suggest to make this configurable as well. As agreed on the chat, we'll use option groups to put relevant options nicely together in the future. As for now there are not that many options for the `podman` plugin so I'd say adding a few more would not hurt.